api_v3/services/UserEntryService.php

```diff
@@ -94,10 +95,6 @@ public function deleteAction($id)
 	 */
 	public function listAction(KalturaUserEntryFilter $filter, KalturaFilterPager $pager = null)
```

Comment:
Why force the passing of the filter?
We should stick to the standards where we allow null in the function signature and create an empty filter if one was not passed.

I suggest reverting the if statement deletion and adding null defaul value tot the filter.